src/content/examples/cors-header-proxy.md

```diff
@@ -132,8 +132,12 @@ function handleOptions(request) {
     // Handle CORS pre-flight request.
     // If you want to check the requested method + headers
     // you can do that here.
+    let respHeaders = corsHeaders;
```

Comment:
Introducing `respHeaders` like this can be a little deceptive, since the subsequent code still mutates the one object that underlies both `respHeaders` and `corsHeaders`.  It might be better to do something that creates a fresh object, like:
```
let respHeaders = {
  ...corsHeaders,
  "Access-Control-Allow-Headers": request.get("Access-Control-Request-Headers"),
}
```
...in case someone later adds code that tries to reuse `corsHeaders` and incorrectly assumes that it contains the values it was originally declared with.